src/bundler.js

```diff
@@ -98,7 +98,7 @@ export async function bundleProd({
 			htmlEntriesPlugin({ cwd, publicDir, publicPath: '/' }),
 			publicPathPlugin({ publicPath: '/' }),
 			aliasesPlugin({ aliases, cwd: root }),
-			htmPlugin(),
+			htmPlugin({ production: true }),
```

Comment:
By default this is set to true inside here https://github.com/preactjs/wmr/blob/3fbed71240cd1ae252c2af68192b9b0fb0cad4aa/src/plugins/htm-plugin.js#L13. so you

Reply:
(heh, yes you are right)